### PR TITLE
Add detailed methods for simupop

### PR DESCRIPTION
**Pull request by ashander**
Unfortunately discover a probable bug from #32

### CODE REVIEW

File: forwards_paper.tex [hunk 1]
```diff
@@ -1043,33 +1046,81 @@ \section*{Discussion}
...
-All of the above installed via conda, except for msprime, which must be installed from GitHub.
-
-\subsection{Requirements for the \simupop{} implementation}
-
-The simulations are implemented the Python package \ftprime{} 0.0.6rc0
-which is available at \url{https://www.github.com/ashander/ftprime}.
+\subsection*{The \simupop{} implementation}
+
+We implemented simulations in a Python script, usable as a command-line utility,
+taking input parameters for population size $N$ and scaled recombination $\rho$.
```
Comment (jeromekelleher) [inline forwards_paper.tex]: recombination rate

File: forwards_paper.tex [hunk 2]
```diff
@@ -1043,33 +1046,81 @@ \section*{Discussion}
...
+As described in 'Simulation benchmarks', we set scaled mutation $\theta=\rho$,
```
Comment (jeromekelleher) [inline forwards_paper.tex]: set the scaled mutation rate

File: forwards_paper.tex [hunk 3]
```diff
@@ -1043,33 +1046,81 @@ \section*{Discussion}
...
+we used simplify interval of 1000 and simulated for $T=10N$ generations in each
+simulation, introducing deleterious mutations at a rate of $\rho/100$ per
+generation.
+Because \simupop{} requires explicit locations for loci, we set the number of
+loci $n=\rho / (4Nr)$ and marked
+$k={\rm round}(\mu / (100 (\mu + \mu / 100))) = 0$ of these as selected loci.
+\jda{unfortunately probably what we did...}
+The selected loci were chosen as a random sample of all the loci.
+
+We coupled \simupop{} to \msprime{} using a Python class \texttt{RecombCollector} 1)
```
Comment (jeromekelleher) [inline forwards_paper.tex]: that 1)

File: forwards_paper.tex [hunk 4]
```diff
@@ -1043,33 +1046,81 @@ \section*{Discussion}
...
+tracks time, and 2) collects and parses recombination information provided by
+the \texttt{Recombinator} class of \simupop{}, adding them to a \tnodetable{} and
+\tedgetable{} upon which simplification is performed every 1000 years (using
```
Comment (jeromekelleher) [inline forwards_paper.tex]: Can we phrase this in generations? Years is confusing, as it sounds a bit like the simulator time.

File: forwards_paper.tex [hunk 5]
```diff
@@ -1043,33 +1046,81 @@ \section*{Discussion}
...
+a mapping of these IDs to the \tnodetable IDs is passed to
+an instance of \texttt{RecombCollector} and used internally to update the tables.
+The \texttt{simuPOP.Population()} is then evolved.
+Before mating: selected mutations are created (using
+\texttt{simuPOP.SNPMutator}) and multiplicative selection occurs (using
+\texttt{simuPOP.PyMlSelector}) only on the selected loci;
+the internal time of the \texttt{RecombCollector} instance is updated.
+During mating: male and female parents are drawn randomly, recombination data is
+output by \simupop{} to the \texttt{RecombCollector} instance.
+After mating: nothing happens 999/1000 years, every 1000 years the tables are
```
Comment (jeromekelleher) [inline forwards_paper.tex]: Again, years is a bit confusing here. Is this a simupop thing?

**Review by jeromekelleher — APPROVED** [review]
Looks good. I'm not sure if this level of detail belongs in a Methods section or an appendix/supplement. We can see how the overall section looks though and decide where it should live afterwards.

File: forwards_paper.tex (hunk 4 above)
Comment (ashander) [reply forwards_paper.tex]: Thanks. Good call it's a tick from working with overlapping generations.

**Pull request merged**